bin/sapconf

```diff
@@ -92,8 +93,7 @@ stop() {
       log "no active sapconf tuning - profile=$profile -, so nothing to revert"
       exit 0
```

Comment:
here we need to remove the sapconf_act_profile file, as it does not contain a valid sapconf profile